src/main/java/io/confluent/connect/jdbc/JdbcUtils.java

```diff
@@ -92,16 +92,20 @@ protected SimpleDateFormat initialValue() {
     DatabaseMetaData metadata = conn.getMetaData();
     ResultSet rs = metadata.getTables(null, null, "%", null);
     List<String> tableNames = new ArrayList<>();
-    while (rs.next()) {
-      if (types.contains(rs.getString(GET_TABLES_TYPE_COLUMN))) {
-        String colName = rs.getString(GET_TABLES_NAME_COLUMN);
-        // SQLite JDBC driver does not correctly mark these as system tables
-        if (metadata.getDatabaseProductName().equals("SQLite") && colName.startsWith("sqlite_")) {
-          continue;
+    try {
```

Comment:
Excellent point... I may be somewhat stuck in the mode of writing JDK6 compatible code. In fact, even more of this can be simplified since `ResultSet` is automatically closed when its statement is closed, so I've updated to handle all the `try` blocks to use try with resources on the `Statement` where one is explicitly created and on the `ResultSet` for the ones returned by `DatabaseMetadata` calls.